rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -822,11 +822,11 @@ void spu_thread::do_dma_transfer(const spu_mfc_cmd& args)
 					continue;
 				}
 
-				res->fetch_and(~1ull);
+				res->operator ++();
```

Comment:
it was a design choice since res is pointer, making direct incrementation needing dereferencing. not that it matters since later commits remove those lines.